### PR TITLE
Happy-API - Anna Hellqvist

### PR DESCRIPTION
**Pull request by AnnaHellqvist**
View the API:
https://annas-thoughts-api.herokuapp.com/

View the frontend:
https://annas-happy-thoughts.netlify.app/

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -18,9 +18,69 @@ const app = express()
...
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140
+  },
+  name: {
+    type: String,
+    default: 'Anonymous'
```
Comment (evelinabe) [inline server.js:32]: Nice that you added more user info!

File: server.js [hunk 2]
```diff
@@ -18,9 +18,69 @@ const app = express()
...
+
+  try {
+    await Thought.updateOne({ _id: req.params.thoughtId }, { $inc: { hearts: 1 } });
+    res.status(200).json();
+
+  } catch (err) {
+    res.status(400).json({
+      message: 'Could not save like, thought not found',
+      error: err.errors
+    })
```
Comment (evelinabe) [inline server.js:81]: Like your error handling!

**Review by evelinabe — APPROVED** [review]
Congratulations on another project really well done! I like that you added the extra functionality of adding user name and counting likes of that user! Your code is clean, although you could delete commented code that you yourself did not write.

**Review by KarolCislowski — APPROVED** [review]
Great project 💯

**Pull request closed without merging**